tests/test_siamese_spectra_model.py

```diff
@@ -125,17 +125,17 @@ def test_model_training(simple_training_spectra):
                                     intensity_scaling=0.5, base_dims=(200, 200),
                                     embedding_dim=100,
                                     train_binning_layer=False,
-                                    same_prob_bins=np.array([(0, 1.0)]),
+                                    same_prob_bins=np.array([(0, 0.5), (0.5, 1.0)]),
                                     average_pairs_per_bin=20,
                                     batch_size=2,
                                     num_turns=20,
                                     )
-    scp_train, _ = select_compound_pairs_wrapper(simple_training_spectra, settings)
+    scp_train = select_compound_pairs_wrapper(simple_training_spectra, settings)
 
     # Create generators
     train_generator_simple = DataGeneratorPytorch(spectrums=simple_training_spectra, selected_compound_pairs=scp_train,
                                                   settings=settings)
-
+    settings.same_prob_bins = np.array([(0, 1.0)])
```

Comment:
The validation loss calculator does not work for such a small dataset with two bins. So I set it here to 1 bin. 
I did not really check if it makes sense that it doesn't work with two bins, but since I noticed that some changes need to be made anyway, I will have a look at this in a different PR.